backend/user-service/routes/auth.js

```diff
@@ -0,0 +1,63 @@
+//user-service/backend/routes/auth.js
+import cookie from 'cookie';
+import dotenv from 'dotenv';
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import { getUserByEmail, getUserCompleteByEmail, getUserPasswordByEmail, insertUser, hashPassword, validatePassword, updateUserName } from '../database.js';
```

Comment:
i think getUserPasswordByEmail is not an export from database.js so this might throw an import error

Reply:
Oh yeah nice I was deleting functions on the github website I forgot to check the dependencies